src/main/java/org/fundacionjala/coding/manuel/movies/Movie.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.manuel.movies;
+
+/**
+ * Created by Juan Manuel on 3/18/2018.
+ */
+public abstract class Movie {
+    public static final int REGULAR = 0;
+    public static final int NEW_RELEASE = 1;
+    public static final int CHILDRENS = 2;
```

Comment:
These static variables are no longer sense if you are using inheritance.
Please remove them. and take advantage of inheritance.